.github/workflows/publish_precompiled.yml

```diff
@@ -0,0 +1,86 @@
+name: Publish Precompiled
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  ELIXIR_OPTS: "--warnings-as-errors"
+  ERLC_OPTS: "warnings_as_errors"
+  LANG: C.UTF-8
+  S3_BUCKET: s3.hex.pm
+
+
+jobs:
+  publish_precompiled:
+    strategy:
+      fail-fast: true
+      # Prevent each job edit builds.txt concurrently.
+      max-parallel: 1
+      matrix:
+        include:
+          - otp: 24
+            otp_version: 24.3
+          - otp: 25
+            otp_version: 25.0
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 50
+
+      - uses: erlef/setup-beam@v1
+        with:
+          otp-version: ${{ matrix.otp_version }}
+          version-type: strict
+
+      - name: Build Elixir Release
+        run: |
+          make Precompiled.zip
+          mv Precompiled.zip main-otp-${{ matrix.otp }}.zip
```

Comment:
repo.hex.pm also has main.zip (without -otp-X) which is compiled against oldest supported OTP. We should support it as well and then I think we can ship it.

Reply:
@wojtekmach So in this case it should build with otp24 right?

Reply:
Yes

Reply:
Let me try

Reply:
@wojtekmach main.zip added in https://github.com/elixir-lang/elixir/pull/12118/commits/6a2c7d5dd75fdd3d93ee7734920c531dc313601f. Please kindly review.